src/SFA.DAS.Commitments.Application/Commands/UpdateApprenticeshipStopDate/UpdateApprenticeshipStopDateCommandHandler.cs

```diff
@@ -118,18 +118,8 @@ private void ValidateChangeDateForStop(DateTime newStopDate, Apprenticeship appr
             if (newStopDate.Date > _currentDate.Now.Date)
                 throw new ValidationException("Invalid Date of Change. Date cannot be in the future.");
 
-            if (newStopDate.Date >= apprenticeship.StopDate)
-                throw new ValidationException("Invalid Date of Change. Date must be before current stop date.");
-
             if (newStopDate.Date < apprenticeship.StartDate.Value.Date)
-                throw new ValidationException("Invalid Date of Change. Date cannot be before the training start date.");
-
-            if ( apprenticeship.PaymentStatus != PaymentStatus.PendingApproval && 
-                _academicYearValidator.Validate(newStopDate.Date) == AcademicYearValidationResult.NotWithinFundingPeriod)
-            {
-                throw new ValidationException("Invalid Date of Change. Date cannot be before the academic year start date.");
-            }
-            
```

Comment:
Again payments calc will be affected by being able to enter a stop date into the previous year (the calc for the previous year has been already run and we ca't claim money back from it)